### PR TITLE
Added stars to mark favorite items (#2102)

### PR DESCRIPTION
**Pull request by Aga-C**
Hi,

I've added marking favorite items with stars, so it's visible which items from a particular album are added to favorites. I've added it in both grid view and list view in the top-right corner. In case of grid view, I had to move video duration to the bottom-right corner of the thumbnail.

image

### CODE REVIEW

**Comment by tibbi** [comment]
try adding more padding to the star, looks too big. I just added a new string mark_favorite_items, please add a new toggle into the app settings -> File thumbnail style for this pull request

**Comment by tibbi** [comment]
guess it can be enabled by default, just test it properly with other thumbail items like file type, video duration, selection check etc, to make sure they dont overlap

**Comment by Aga-C** [comment]
I've added a setting, made star smaller, and also moved everything on a thumbnail around to not overlap anything.

image

File: app/src/main/res/layout/photo_item_grid.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_alignParentStart="true"
-        android:layout_alignParentTop="true"
+        android:layout_alignParentBottom="true"
```
Comment (tibbi) [inline app/src/main/res/layout/photo_item_grid.xml:20]: you cant just change something like this, it now fully overlaps with the file name

**Comment by Aga-C** [comment]
Should I move everything above the file name if it's visible?

**Comment by tibbi** [comment]
no, that would hide the image then. Keep icons at the top, not bottom. Lets move the video duration to the right of the Video icon and Favorite at the top right corner. Make the Video and Favorite icons the same size and margins though so they are symmetric.

**Comment by Aga-C** [comment]
I've changed it.

image

**Comment by tibbi** [comment]
ye should be good, Ill just remove the new dimens values, I want to avoid adding useles new values like that. Ill do that in master though, it is easier to edit multiple files like that. Thanks

**Pull request merged**